modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/JavaMapBuilderGenericsCheck.java

```diff
@@ -34,15 +34,17 @@ protected String doProcess(
 			String fileContent)
 		throws IOException {
 
-		return _formatGenerics(javaTerm, fileContent);
+		return _formatGenerics(fileName, javaTerm, fileContent);
```

Comment:
@TamyrisTorres, is it very costly to sort all parameters alphabetically?

If not, please do it to mitigate the risk of Brian complain about it.